app/code/Scandiweb_Test/etc/module.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0"?>
+<config
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
+	<module name="Scandiweb_ScandiwebTest"></module>
```

Comment:
Use `<module name="Scandiweb_ScandiwebTest"/>`, if the contents are empty.

Please replace the above line with `<module name="Scandiweb_ScandiwebTest"></module>`